docs/program-setup/mbed_targets.md

```diff
@@ -194,7 +194,9 @@ Enabling `is_disk_virtual` adds delay after flashing firmware binary to make sur
 
 ### supported_toolchains
 
-The `supported_toolchains` property is the list of toolchains that support a target. The allowed values for `supported_toolchains` are `ARM`, `uARM`, `ARMC5`, `ARMC6`, `GCC_ARM` and `IAR`.
+The `supported_toolchains` property is the list of toolchains that support a target. The allowed values for `supported_toolchains` are `ARM`, `ARMC5`, `UARM`, `ARMC6`, `GCC_ARM` and `IAR`.
```

Comment:
We should probably also mention `supported_c_libs` as it is also related to toolchain support.

Reply:
Is that related to the Bare Metal question I had?

Reply:
The uArm question. To my understanding, from 6 we don't support uArm.